docarray/typing/tensor/ndarray.py

```diff
@@ -116,6 +117,8 @@ def validate(
             return cls._docarray_from_native(value)
         elif isinstance(value, NdArray):
             return cast(T, value)
+        elif isinstance(value, torch.Tensor):
+            return cast(T, value)
```

Comment:
you should rather import torch here.

be sure to use our utils `import_library`.

Reply:
you can take inspiration here : https://github.com/docarray/docarray/blob/b3649b4b0f020887397f96e757711f71095b7fee/docarray/array/doc_vec/doc_vec.py#L122